### PR TITLE
feat: add GitHub Action for Continuous Deployment

### PR DESCRIPTION
**Pull request by lucasvuotto**
Additional configuration steps will be needed before merging, which I'll follow up in private. Also, a tentatively deploy strategy is presented, but needs approval of the team before merging.

The tentatively deploy strategy is:
- Pushes to tags in the form of `vX.Y.Z` are deployed to all the environments
- Pushes to tags in the form of `vX.Y.Z-additional-data` are deployed just to staging environtment

### CODE REVIEW

**Review by jurajpiar — APPROVED** [review]
LGTM

**Comment by rafaiovlabs** [comment]
bout this PR, basically team must be ok with the deployment scheme proposed:
The tentatively deploy strategy is:

Pushes to tags in the form of vX.Y.Z are deployed to all the environments
Pushes to tags in the form of vX.Y.Z-additional-data are deployed just to staging environment

If ok I can then do some configurations on the Repo.

File: .github/workflows/staging.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+name: RIF Marketplace UI deploy
+
+on:
+  push:
+    tags:
```
Comment (artem-brazhnikov) [inline .github/workflows/staging.yml:5]: I think as we agreed, we have to connect staging area to a branch

**Review by 0xartem — CHANGES_REQUESTED** [review]

File: .github/workflows/staging.yml (hunk 1 above)
Comment (rafaiovlabs) [reply .github/workflows/staging.yml:5]: Yes, at the moment the "staging" branch is deployed to the "staging" bucket.

**Comment by rafaiovlabs** [comment]
We are closing this in favor of PR #745

**Pull request closed without merging**